daemon/health.go

```diff
@@ -156,6 +157,7 @@ func monitor(d *Daemon, c *container.Container, stop chan struct{}, probe probe)
 			ctx, cancelProbe := context.WithTimeout(context.Background(), probeTimeout)
 			results := make(chan *types.HealthcheckResult)
 			go func() {
+				healthCheckCounter.Inc()
```

Comment:
Thoughts on breaking this up into healthy & unhealthy result counters? You can still get the current value, but also get a little more detailed info if you want it. This metric/comment is kinda moot though if you add health status counters to each container because you can just aggregate.